src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -6663,7 +6663,11 @@ void ObjectMgr::SetHighestGuids()
 
     result = CharacterDatabase.Query("SELECT MAX(setguid) FROM character_equipmentsets");
     if (result)
-        _equipmentSetGuid = (*result)[0].GetUInt64()+1;
+        _equipmentSetGuid = (*result)[0].GetUInt64();
```

Comment:
you need the `+1` if your next db query has an empty resultset

Reply:
@roc13x